keywords.robot

```diff
@@ -605,8 +620,12 @@ Prepare To PiKVM Connection
     Remap Keys Variables To PiKVM
     Open Connection And Log In
     ${platform}=    Get Current RTE Param    platform
-    ${pikvm_ip}=    Get Current RTE Param    pikvm_ip
-    Set Global Variable    ${PIKVM_IP}
+    IF    '${SNIPEIT}' == 'yes'
+        ${pikvm_ip}=    Snipeit Get PiKVM IP    ${RTE_IP}
+        Set Global Variable    ${PIKVM_IP}
+    END
+    # If snipeit is set to "no", fetch it from command line
+    Variable Should Exist    ${PIKVM_IP}
```

Comment:
add `msg=...` Explaining this in the error message

Reply:
Done 271316b396eccc0c67738c27e883e9bbf6eeeb54